blog/2023-06-14-paralus-at-platformcon-2023/index.md

```diff
@@ -0,0 +1,113 @@
+---
+slug: paralus-platformcon-2023
+title: "Paralus at PlatformCon - 2023"
+authors: [saim]
+tags: [paralus]
+image: /img/docs/paralus-platformcon-2023-speaker-banner.jpeg 
+---
+
+With over 150 speakers and 20,000 attendees [PlatformCon 2023](https://platformcon.com/), the virtual conference created by and for platform engineers was a hub of activity and excitement for tech enthusiasts and industry professionals alike. Team Paralus was also a part of such a vibrant community for the second edition of PlatformCon.
+
+This event brought together a global community of techies, developers, and platform engineers to share the latest advancements and trends in the field of platform engineering. 
+
+The event also featured a diverse range of talks ranging from platform-culture, platform-blueprint, platform-impact to platform-stories. You can watch all the sessions on the  [Platform Engineering YouTube Channel](https://www.youtube.com/@PlatformEngineering).  
+
+<!--truncate -->
+
+In this blog post, we'll be sharing our experience of interacting with attendees at [PlatformCon 2023](https://platformcon.com/) virtually, some glimpses of our talk - [Enable secure self-service access to Kubernetes clusters with Paralus](https://www.youtube.com/watch?v=UbLnCGOv0Rk). where we spoke about how Paralus removes the cognitive load from platform teams working on repetitive tasks including setting up RBAC, revoking permissions, pre-configuring roles across clusters, namespaces, projects, and more.
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/UbLnCGOv0Rk" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
+
+During [PlatformCon 2023](https://platformcon.com/), we were inundated with visitors reaching out to us on the Platform Engineering community [slack channel](https://platformengin-b0m7058.slack.com/join/shared_invite/zt-1vsqjlb51-SiDy9cp2EBGffIUj5_g2mA#/shared-invite/email), who were keen to learn more about our project. We had some quick demos on how to use the project, along with some tasty cookies and cool stickers to give away. The visitors were curious about the project and its potential, and some of them even reached out for a dedicated demo of the tool post PlatformCon.   
```

Comment:
May be add some spacing here, in preview mode this para immediately starts after the video thumbnail